tests/common/tests/kalman_fitting_wire_chamber_test.hpp

```diff
@@ -67,6 +67,7 @@ class KalmanFittingWireChamberTests : public KalmanFittingTests {
         vecmem::host_memory_resource host_mr;
 
         detray::wire_chamber_config wire_chamber_cfg;
+        wire_chamber_cfg.mapped_material(detray::vacuum<scalar>());
```

Comment:
@niermann999  2 GeV test of wire chamber still fails with (berylium + 100 um thickness) setup so I decided to make portal materials empty :/

Reply:
Ok